cmd/crypto/kms.go

```diff
@@ -111,6 +114,11 @@ type masterKeyKMS struct {
 // to the generated keys.
 func NewKMS(key [32]byte) KMS { return &masterKeyKMS{masterKey: key} }
 
+// Returns error as KMS is configured directly using master key
+func (kms *masterKeyKMS) KMSInfo() (endpoint, name, authtype string, err error) {
```

Comment:
Having a master key configured instead of a KMS like Vault is not an error. Instead we should provide proper values where applicable and empty string otherwise. e.g (including the comment above about `KMSInfo` type),
```go
func (kms *masterKeyKMS) Info() (info KMSInfo, err error) {
  return KMSInfo {
     Endpoint: "",
     Name:"",
     AuthType: "master-key" // need to consult this with folks who have worked on MinIO SSE implementation
  }, nil
}
```

Reply:
@aead . Can you please confirm , `AuthType: "master-key"` is good to go for the MNSInfo ?